docs/diagrams/FindTagSequenceDiagram.puml

```diff
@@ -0,0 +1,77 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":FindTagCommandParser" as FindTagCommandParser LOGIC_COLOR
+participant "p:TagContainsKeywordPredicate" as TagContainsKeywordPredicate LOGIC_COLOR
+participant "f:FindTagCommand" as FindTagCommand LOGIC_COLOR
+participant "result:CommandResult" as CommandResult LOGIC_COLOR
+
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("findtag friends")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("findtag friends")
+activate AddressBookParser
+
+create FindTagCommandParser
+AddressBookParser -> FindTagCommandParser
+activate FindTagCommandParser
+
+FindTagCommandParser --> AddressBookParser
+deactivate FindTagCommandParser
+
+AddressBookParser -> FindTagCommandParser : parse("friends")
+activate FindTagCommandParser
+
+create TagContainsKeywordPredicate
+FindTagCommandParser -> TagContainsKeywordPredicate
+activate TagContainsKeywordPredicate
+
+TagContainsKeywordPredicate --> FindTagCommandParser : p
```

Comment:
I think you forgot to deactivate TagContainsKeywordPredicate here!

Reply:
Yup my bad, totally missed out on that. Thanks for spotting it! 😄